src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+function Calculator() {
+  return (
+    <div className="wrapper">
```

Comment:
Please be sure to make the necessary changes as requested by the previous reviewer 😉 Your calculator function should be modified so that your Calculator component meets the requirements 📝App.js is class-based as you commented so just replicate what you did with this component 🤓 

![image](https://user-images.githubusercontent.com/15807343/188720084-5f661e01-18df-48c7-8741-d3d2aea93ceb.png)